### DISCUSSION TITLE
Introduce edge docs

### DISCUSSION THREAD

**Discussion by bopm** [tailwindlabs/tailwindcss#14352] [Ideas]
With 4-alpha's active development, it's odd not to see it in the version select dropdown on the Tailwind site. Can we get either v4 in that dropdown or separate https://edge.tailwindcss.com/docs if, for some reason, it's problematic to add v4 to the dropdown?

**Comment by adamwathan** [comment]
There are currently no v4 docs to share I'm afraid 🫣 Once they exist they will be on a website somewhere!
